README.md

```diff
@@ -107,6 +108,33 @@ unit of work and continuity.
   ![](/docs/img/code.png)
 
 
+## Airflow Versions, backporting packages and future upgrade path
+
+
+Currently stable versions of Apache Airflow are released in 1.10.* series. We are working on the
+future, incompatible version of Airflow from the 2.0.* series. It is going to be released in
```

Comment:
Let's not say "incompatible" like this please -- that's a stronger word than we need.

```
Currently stable versions of Apache Airflow are released in 1.10.* series. We are working on the
future, Airflow 2.x series which has some backwards-incompatible changes

```